bucket/panwriter.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "0.6.5",
+    "description": "Markdown editor with pandoc integration and paginated preview",
+    "homepage": "https://panwriter.com",
+    "license": "GPL-3.0-only",
+    "depends": "pandoc",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/mb21/panwriter/releases/download/v0.6.5/PanWriter-Setup-0.6.5.exe#/dl.7z",
+            "hash": "66275b2ccf8981beb4d42ca0e2655c5f667db1b2df6217fefcfbd2e9dc726c4b",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\", \"$dir\\Uninstall*\" -Recurse"
+            ]
+        }
+    },
+    "bin": "PanWriter.exe",
```

Comment:
Even though, there is command line support for passing file as argument, the binary is completely unusable from terminal

```suggestion
```

![A](https://i.imgur.com/eRbNsEb.png)